flutter/pubspec.lock

```diff
@@ -28,7 +28,7 @@ packages:
       name: archive
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "3.1.6"
+    version: "3.1.2"
```

Comment:
Please, revert changes in `pubspec.lock`.

Reply:
We have reverted changes in `pubspec.lock`.